README.md

```diff
@@ -0,0 +1,30 @@
+# 🛠️ Lowdefy Tutorial
+
+![lowdefy logo](https://lowdefy.com/images/logo.svg)
+
+Welcome to the Lowdefy [Tutorial](<(https://docs.lowdefy.com/tutorial-start)>) repository! Lowdefy is an open-source, low-code framework for building web apps, dashboards, and more. This repository serves as your starting point to explore and build with Lowdefy.
```

Comment:
looks like the link has been commented out here? or not rendering as it should? 
If we want something with a little tool tip, can use: `[Lowdefy Tutorial](https://docs.lowdefy.com/tutorial-start "click here to follow our our guided tutorial") `

would be nice to have something near the beginning for people to click on rather than only at the bottom?

Reply:
that link is broken, will fix in the next commit

Reply:
201e34e5977292aefbaf0e7f7421384b12f019d6